feature/announcement/src/main/java/io/github/droidkaigi/confsched2020/announcement/ui/viewmodel/AnnouncementViewModel.kt

```diff
@@ -57,6 +50,22 @@ class AnnouncementViewModel @AssistedInject constructor(
         )
     }
 
+    fun load() {
+        viewModelScope.launch {
+            try {
+                announcementRepository.refresh()
```

Comment:
Thanks! 😉 
When you open it for the second time or later, you have to wait to read the acquisition from the API.
I think that Fragment is regenerated by ConfigrationChange, but I'm worried why the language is not adapted.
I think View could solve this problem. 👀

Reply:
> I think View(Inside Fragment) could solve this problem. 👀

What do you mean?

Reply:
I'm thinking if the language can be switched by redisplaying in View instead of in ViewModel. 👀

Reply:
Please let me know if you have any problems doing it. 🙏

Reply:
I understand. but I have no idea to redisplay only using View(Inside Fragment)😭 I'm going to think another solution.